src/components/reference-book/page.cjsx

```diff
@@ -81,7 +81,7 @@ module.exports = React.createClass
   render: ->
     {courseId} = @context.router.getCurrentParams()
     # read the id from props, or failing that the url
-    cnxId = @getCNXId()
+    cnxId = @getCnxId()
```

Comment:
Oops, I meant that `Cnx` was correct since `CNX` does not stand for anything else.